projects/ng-imgix/src/lib/ix-srcset.directive.ts

```diff
@@ -1,5 +1,6 @@
 import { Directive, Input, ElementRef, OnInit } from '@angular/core';
 
+const PKG_VERSION = require('../../package.json').version;
```

Comment:
Are you sure that this works in the browser? As I mentioned in Gatsby, this doesn't "just work" in the browser, it needs a relevant Webpack or Babel build step to transform this into the right value. If that's working here, then go ahead 👍

Reply:
I mean it works when I execute the tests (which run in the browser) but that might not mean it'll work on _all_ browsers? I'm willing to keep it for now and catch any edge cases through better testing -- hoping cypress will help there.

Reply:
This isn't a browser thing, if it works in one it'll work in all of them. When you say "execute the tests" are these integration tests, or tests that run on the built library? One way to test if this works is if this `require` still exists in the dist file. If it does, I'd strongly suggest removing this.

Reply:
Ah thanks for clarifying, I now see what you mean. Looks like tests were not running on the built version, so I'm now running into the issue you mentioned whenever I try to import the directive in a test app. I'm doing a bit more research, but I'll either open a new PR with a potential fix or just roll this back altogether. Thanks again @frederickfogerty

Reply:
Resolved offline